clients/cloud/java-springboot/src/main/java/io/confluent/examples/clients/cloud/springboot/ProducerExample.java

```diff
@@ -0,0 +1,53 @@
+package io.confluent.examples.clients.cloud.springboot;
+
+import org.apache.kafka.clients.admin.NewTopic;
+import org.apache.kafka.clients.producer.RecordMetadata;
+import org.springframework.boot.context.event.ApplicationStartedEvent;
+import org.springframework.context.event.EventListener;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.stereotype.Component;
+
+import io.confluent.examples.clients.cloud.DataRecordAvro;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+
+import static java.lang.System.out;
+
+@Log4j2
+@Component
+@RequiredArgsConstructor
+public class ProducerExample {
+
+  private final KafkaTemplate<String, DataRecordAvro> producer;
+  private final NewTopic topic;
+
+  @EventListener(ApplicationStartedEvent.class)
+  public void produce() {
+    // Produce sample data
+    final long numMessages = 10L;
+    for (long i = 0L; i < numMessages; i++) {
+      String key = "alice";
+      DataRecordAvro record = new DataRecordAvro(i);
+
+      log.info("Producing record: {}\t{}", key, record);
+      producer.send(topic.name(), key, record).addCallback(
+          result -> {
+            final RecordMetadata m;
+            if (result != null) {
+              m = result.getRecordMetadata();
+              log.info("Produced record to topic {} partition {} @ offset {}",
+                       m.topic(),
+                       m.partition(),
+                       m.offset());
+            }
```

Comment:
Only for your consideration @gAmUssA, I thought handling the null case might be nice, but just a suggestion.

```
log.info(
              Optional.of(result)
                .map(r -> r.getRecordMetadata())
                .map(m -> String.format("Produced record to topic {} partition {} @ offset {}",
                  m.topic(), m.partition(), m.offset()))
                .orElse("produce.send result was null"));
```

Reply:
I was following java example.

Reply:
No worries... I'm good w/ this but I'll wait on the README before approval.

Reply:
@rspurgeon working on it